crates/sequencer/Cargo.toml

```diff
@@ -76,7 +76,7 @@ use_self = "allow"
 name = "sequencer"
 version = "0.1.0"
 edition = "2021"
-rust-version = "1.76"
+rust-version = "1.79"
```

Comment:
Seems like it's a better practice to put the rust version here. I observed that the rust-toolchain file overrides almost everything. And seems like a good place to put this

https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field

Reply:
Oups my comment is on the sequencer crate but I wanted to comment this also for our high level cargo file

Reply:
Here it's the minimal version required, meaning that the user can only compiler with higher version of rust:

> The rust-version field is an optional key that tells cargo what version of the Rust language and compiler your package can be compiled with. If the currently selected version of the Rust compiler is older than the stated version, cargo will exit with an error, telling the user what version is required.